Dockerfile

```diff
@@ -0,0 +1,49 @@
+ARG PY_VERSION=3.9
+
+FROM amazon/aws-lambda-python:$PY_VERSION
+
+# Declare it a second time so it's brought into this scope.
+ARG PY_VERSION=3.9
```

Comment:
Do we want `PY_VERSION` to be a build variable?  Or does the `3.9` act as a default value if it's not specified as a build variable?  I'm a little confused as to how this lines up with the Python 3.7, 3.8, and 3.9 `Pipfile`s if `3.9` is hardcoded.

Reply:
So that value is used as a default value. So if you provide no `PY_VERSION` build argument it will default to using `3.9`. Per the [documentation](https://docs.docker.com/engine/reference/builder/#arg):

```dockerfile
ARG <name>[=<default value>]
```

Reply:
Perfect!

Reply:
I force-pushed because there was a build arg that wasn't removed in the GitHub Actions workflow when I redid artifact generation but just for additional reference the workflow builds each Lambda version like so:

https://github.com/cisagov/skeleton-aws-lambda-python/blob/6f98ec54b5f137f704612b6d5eb1df5a87411e8f/.github/workflows/build.yml#L153-L157